tc-tests-utils.sh

```diff
@@ -806,6 +896,82 @@ maybe_ssl102_py37()
     esac
 }
 
+maybe_numpy_min_version_winamd64()
+{
+    local pyver=$1
+
+    if [ "${OS}" != "${TC_MSYS_VERSION}" ]; then
+        return;
+    fi
+
+    # We set >= and < to make sure we have no numpy incompatibilities
+    # otherwise, `from deepspeech.impl` throws with "illegal instruction"
+    case "${pyver}" in
+	3.5*)
```

Comment:
mixing spaces and tabs around this block

Reply:
fixes